contracts/cw721-base/src/bin/schema.rs

```diff
@@ -0,0 +1,11 @@
+use cosmwasm_schema::write_api;
+
+use cw721_base::{ExecuteMsg, InstantiateMsg, QueryMsg};
+
+fn main() {
+    write_api! {
+        instantiate: InstantiateMsg,
+        execute: ExecuteMsg,
+        query: QueryMsg,
```

Comment:
Can you do this instead (which is the same that was generated before)?:

```suggestion
        query: QueryMsg<Empty>,
```